### ISSUE TITLE
Skipping python 3.6 still creates CI job in multi-output recipe

### ISSUE THREAD

**Issue by h-vetinari** [conda/conda-build#4206]
In https://github.com/conda-forge/modin-feedstock/pull/13, I updated `skip: true  # [py<37]`, since upstream isn't compatible with 3.6 anymore.

While [trying](https://github.com/conda-forge/modin-feedstock/pull/14) to turn this recipe into a multi-output version, rerendering cause the 3.6 CI jobs to reappear (even though the build is skipped, so CI should stay green).

At first I thought this was a `conda smithy` [issue](https://github.com/conda-forge/conda-smithy/issues/1460), but was advised there that this is actually coming from `conda render` under the hood, and to raise an issue here.

**Comment by leofang** [comment]
Can we please reopen this issue? We hit this issue in skipping PY38 in a conda-forge feedstock. I think it is a waste of the CI resources to launch pipelines and yet do nothing.

**Comment by h-vetinari** [comment]
Given that this was reopened, I updated the OP and the title. Relevant snippet:

> The - somewhat annoying - workaround is to add `skip: true # [something]` to _every single output_.

**Comment by leofang** [comment]
Even adding skip to every single output does not work.

**Comment by h-vetinari** [comment]
Every single output, including the "global" one. I've never seen CI jobs created once _every possible output_  was skipped.

**Comment by jakirkham** [comment]
Yes that is what Leo had tried earlier. Reference [`meta.yaml`]( https://github.com/conda-forge/cupy-feedstock/blob/4f9c72e467767c3a53deca65995a390eb1346eb4/recipe/meta.yaml ) for discussion

**Comment by jakirkham** [comment]
Think an important to note is once we [added `python` in the top-level `requirements/host`]( https://github.com/conda-forge/cupy-feedstock/pull/241/commits/616005905071cf5e7659f8b6883b81389353ef23 ), the `skip` (and `conda smithy rerender` in our case) worked

So think it is some combination of top-level requirements and `skip` that make the difference. We could probably poke at that example more to learn what is going on

**Comment by h-vetinari** [comment]
Even though I know how to work around this now, it still feels pointless. A package-level skip should also skip all the outputs IMO.
